src/wallet/wallet_tx_builder.cpp

```diff
@@ -80,45 +85,48 @@ WalletTxBuilder::GetChangeAddress(
 
     auto changeAddressForZUFVK = [&](
             const ZcashdUnifiedFullViewingKey& zufvk,
-            const std::set<ReceiverType>& receiverTypes) {
+            const std::set<ReceiverType>& receiverTypes)
+        -> tl::expected<ChangeAddress, AddressResolutionError> {
```

Comment:
Similarly, this return type should either remain `RecipientAddress` or the body is wrong.

Reply:
The call site for this is in a match expression where each result is a `tl::expected`.